lib/cn/wbt_builder.c

```diff
@@ -471,6 +476,7 @@ wbb_add_entry(
     wbb->cnode_last_klen = klen;
 
     wbb->cnode_nkeys++;
+    wbb->cnode_wlen += key_obj_len(kobj) + vlen;
```

Comment:
I guess not.  I think that's fine.  But "wlen" implies "written data", which would include metadata.  Maybe a new name to avoid confusion with the alen/wlen stats used in many other places with specific meanings.  How about:  `kvlen`?

Reply:
What about dlen for data length?

Reply:
Fine by me, though I like `kvlen` better :)

Reply:
What if we play rock paper scissors for it?